joomla/joomla-cms
# Properly use sublayout

Pull Request for Issue # .

### Summary of Changes
A few cases where layouts were calling `JLayoutHelper::render` when they could have been calling `$this->sublayout` instead have been changed. The main benefit of this is that it makes it easier to override parts of a layout.  

### Testing Instructions
Check that the layouts are still rendering as expected.
The `content.info_block` layout appears at the top of article pages (and others) on the site. 
The `searchtools.default` layout renders the search tools at the top of list pages in the administrator.
The `editors.buttons` layout renders the buttons that appear below the editor (for example, when editing an article).

### Documentation Changes Required
Probably not.
